topcoffea/modules/corrections.py

```diff
@@ -58,8 +58,9 @@ def get_method1a_wgt_doublewp(effA, effB, sfA, sfB, cutA, cutB, cutC):
 # Evaluate btag sf from central correctionlib json
 def btag_sf_eval(jet_collection,wp,year,method,syst):
     # Get the right sf json for the given year
+    is_run3 = year[2] == "2"
```

Comment:
Let's use a more clear check, like:
```suggestion
    is_run3 = yearstartswith("202")
```

Reply:
I would actually suggest checking for exact string agreement. E.g. `if year in ["2022","2022EE",...]`. I think it's safer

Reply:
I am in favor of the `.startswith` solution. Given that Run2 years will always return `is_run3 = False` and future Run3 samples will automatically make this boolean `True`, I think that solution will work in the near future, without the need to re-modify this line (as instead, it would happen with the list solution). I don't see any particular safety issues that the list implementation can prevent and the `.startswith` solution cannot